### PR TITLE
Updated presentations

### PR DESCRIPTION
**Pull request by johannbrehmer**

### CODE REVIEW

File: _data/people/johannbrehmer.yml [hunk 1]
```diff
@@ -6,16 +6,33 @@ institution: New York University
...
 presentations:
+  - title: Normalizing flows and the likelihood ratio trick in particle physics
+    date: 2020-01-06
+    url:
+    meeting: Deep learning seminar
+    meetingurl:
+    location: University of Bremen, Germany
+    focus-area: as
+    project: madminer
+  - title: Mining gold: Improving simulation-based inference with latent information (poster)
```
Comment (henryiii) [inline _data/people/johannbrehmer.yml]:
```suggestion
  - title: "Mining gold: Improving simulation-based inference with latent information (poster)"
```

Colons not allowed in YAML unquoted strings (unless they are not followed by a space, such as a URL)

Comment (henryiii) [reply _data/people/johannbrehmer.yml]: @johannbrehmer, you have disabled the "allow maintainers to push to your PR" option so I can't fix this for you. Please fix it.

Comment (johannbrehmer) [reply _data/people/johannbrehmer.yml]: Thanks @henryiii!

Comment (henryiii) [reply _data/people/johannbrehmer.yml]: :+1:

**Pull request merged**